### ISSUE TITLE
Missing songs in desktop app

### ISSUE THREAD

**Issue by XLave** [MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-#3617]
I have a problem with the content showed in the web version of Google Music and the desktop app, there are some tracks that are missing in the app but they're in the cloud of the web version. Also checked if there were under another name album but not. I'm attaching a screenshot showing the problem. I tried to uninstall and reinstall de app but the problem persists.
![Anotación 2019-08-23 152314](https://user-images.githubusercontent.com/54935959/64328789-636fe180-cfce-11e9-94b7-5175b17342af.jpg)

**Comment by jostrander** [comment]
Is there anything in common with the songs that are gone? Are they disliked or do you have any custom css enabled?

**Comment by XLave** [comment]
No, the all have the same metadata (except for the track title). They share album, artist, etc...

**Comment by XLave** [comment]
I still having the same issue, is there any fix to it??
https://imgur.com/a/tau2S5d

**Comment by mchristen** [comment]
I'm also experiencing this issue. I recently built a new desktop and when I loaded up GPMDP a bunch of songs aren't showing up. They are still showing up when I login to the web version directly using Chrome and using and the Android App. 

It almost seems like it's using an older copy of the library metadata.
